web/handlers/admin/dashboard.go

```diff
@@ -25,36 +28,53 @@ type dashboardHandler struct {
 }
 
 func (h dashboardHandler) overview(w http.ResponseWriter, req *http.Request) (interface{}, error) {
-	roomRef := h.netInfo.RoomID.Ref()
-
-	onlineRefs := h.roomState.List()
+	var (
+		ctx     = req.Context()
+		roomRef = h.netInfo.RoomID.Ref()
+
+		onlineRefs   []refs.FeedRef
+		refsUpdateCh = make(chan []refs.FeedRef)
+		onlineCount  = -1
+	)
+
+	// this is an attempt to sidestep the _dashboard doesn't render_ bug (issue #210)
+	// first we retreive the member state via a goroutine in the background
+	go func() {
+		refsUpdateCh <- h.roomState.ListAsRefs()
+	}()
+
+	// if it doesn't complete in 10 seconds the slice stays empty and onlineCount remains -1 (to indicate a problem)
```

Comment:
Hmm, okay, this is going to make detecting the bug easier for future troubleshooting